src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs

```diff
@@ -84,13 +73,15 @@ protected AbstractGenerateMemberService()
             INamedTypeSymbol containingType,
             TExpressionSyntax expression,
             CancellationToken cancellationToken,
-            out INamedTypeSymbol typeToGenerateIn,
-            out bool isStatic)
+            out INamedTypeSymbol? typeToGenerateIn,
+            out bool isStatic,
+            out bool isColorColorCase)
```

Comment:
Only comment - is there a more general name for this case?

Reply:
The spec describes it thusly: https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11762-identical-simple-names-and-type-names

but we'e found referring to it as the `Color Color case` one of the easiest ways to make it clear what this is referring to.